_posts/blog/2016-04-15-subscribe_on.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "RxJava: subscribeOn vs observeOn"
+modified:
+categories: blog
+author: tomek_polanski
+excerpt:
+tags: [RxJava]
+image:
+date: 2016-04-14T13:00:55-01:00
+---
+
+
+## RxJava: subscribeOn vs observeOn
+
+One of the strongest aspects of RxJava is the simple way to schedule work on a desired thread using either ``subscribeOn`` or ``observeOn``. Understanding how each one works is crucial.  
```

Comment:
> Understanding how each one works is crucial. 

Crucial to what? I would lead the last sentence with something like 

> While they seem simple enough at a glance, understanding how they work is crucial to achieving your desired threading assignment